atat/routes/dev.py

```diff
@@ -183,23 +188,36 @@ def get_or_create_dev_persona(persona):
 
 @local_access_bp.route("/dev-new-user")
 def dev_new_user():
-    first_name = request.args.get("first_name", None)
-    last_name = request.args.get("last_name", None)
-    dod_id = request.args.get("dod_id", None)
+    first_name = request.args.get("first_name", None, type=str)
+    last_name = request.args.get("last_name", None, type=str)
+    dod_id = request.args.get("dod_id", None, type=str)
 
+    # 1: It need to check fist that have all the parameters
     if None in [first_name, last_name, dod_id]:
         raise IncompleteInfoError()
 
+    # This interphase convert dictionaries into Objects instance. It is Require for step 2.
+    class Struct(object):
+        def __init__(self, **entries):
+            self.__dict__.update(entries)
+
+    # 2: Using standard validator to validate inputs or throw a error
+    is_name({}, Struct(**{"data": first_name}))
+    is_name({}, Struct(**{"data": last_name}))
+    is_number({}, Struct(**{"data": dod_id}))
```

Comment:
The `entries` parameter for `Struct.__init__` already has `**` so doing this on both ends makes the code harder to understand. The symbols are confusing so maybe consider:

```suggestion
    is_name({}, Struct(data=first_name))
    is_name({}, Struct(data=last_name))
    is_number({}, Struct(data=dod_id))
```